NOTICE

```diff
@@ -1014,3 +1014,39 @@ BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
 ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
+
```

Comment:
@niwilso is this license taken from the python-gdcm project?

Reply:
@omri374 - This license is taken from [GDCM library](https://github.com/malaterre/GDCM/blob/master/Copyright.txt) since that is the repo which was linked from the [PyPI page](https://pypi.org/project/python-gdcm/).

However, upon further search I do see the GitHub repo for [python-gdcm](https://github.com/tfmoraes/python-gdcm) specifically and found this [Apache 2.0 license](https://github.com/tfmoraes/python-gdcm/blob/master/LICENSE).

Should I replace the text added to the Presidio notice with the python-gdcm Apache 2.0 license instead?

Reply:
Yes please, This text is a custom license and it would be better to have the standard Apache 2.0 license. Thanks!